go/inst/instance.go

```diff
@@ -204,6 +222,21 @@ func (this *Instance) IsNDB() bool {
 	return strings.Contains(this.Version, "-ndb-")
 }
 
+// IsReplicationGroup checks whether the host thinks it is part of a known replication group. Notice that this might
+// return True even if the group has decided to expel the member represented by this instance, as the instance might not
+// know that under certain circumstances
+func (this *Instance) IsReplicationGroupMember() bool {
+	return this.ReplicationGroupName != ""
+}
+
+func (this *Instance) IsReplicationGroupPrimary() bool {
+	return this.IsReplicationGroupMember() && this.ReplicationGroupPrimaryKey == this.Key
```

Comment:
Please use `Equals()`
```suggestion
	return this.IsReplicationGroupMember() && this.ReplicationGroupPrimaryKey.Equals(&this.Key)
```

Reply:
Fixed